receiver/awsecscontainermetricsreceiver/receiver_test.go

```diff
@@ -17,28 +17,38 @@ package awsecscontainermetricsreceiver
 import (
 	"context"
 	"errors"
+	"fmt"
 	"io/ioutil"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/consumer/consumertest"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/ecsutil"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver/internal/awsecscontainermetrics"
 )
 
 type fakeRestClient struct {
```

Comment:
Would it make sense to put this in a `ecsutiltest` package that's shared?

Reply:
The reason this came to mind in particular is that it seems a bit strange for the path variables to be exposed downstream like here

Reply:
I took a shot at this with a `testing.T` in the signature to account for exported test util.